bot/exts/core/help.py

```diff
@@ -153,12 +160,17 @@ def _handle_not_found(self, query: str) -> None:
 
         Will pass on possible close matches along with the `HelpQueryNotFound` exception.
         """
-        # Combine command and cog names
-        choices = list(self._bot.all_commands) + list(self._bot.cogs)
+        # Check if parent command in valid incase subcommand is invalid.
+        if " " in query:
+            parent, sub_command = query.split()
```

Comment:
Don't we have deeper nested subcommands than once? I think you should pass `maxsplit=1`.